repos/ports/run/vbox_win.inc

```diff
@@ -74,7 +74,7 @@ append config_of_app {
 		needs to be enabled in the .vbox file (disabled by default).
 		(Recording is configured to use the external mic.)
 	-->
-	<start name="audio_drv">
+	<start name="audio_drv" priority="-1">
```

Comment:
I'm surprised that a priority downgrade of the audio driver eliminates choppy sound. Did you investigate the interactions of the scenario?